projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/vdk_ui.py

```diff
@@ -163,20 +165,38 @@ def create_deployment(name: str, team: str, path: str, reason: str):
         :param reason: the reason of deployment
         :return: output string of the operation
         """
+        # Make a copy of the directory at the same location
+        destination_path = os.path.join(path, os.path.basename(path))
+        shutil.copytree(path, destination_path)
+
+        # Clear outputs for all notebooks in the destination path
+        for dir_path, _, filenames in os.walk(destination_path):
+            for filename in filenames:
+                if filename.endswith(".ipynb"):
+                    notebook_path = os.path.join(dir_path, filename)
+                    clear_notebook_outputs(notebook_path)
+
         printer = InMemoryTextPrinter()
-        cmd = JobDeploy(RestApiUrlConfiguration.get_rest_api_url(), printer)
-        cmd.create(
-            name=name,
-            team=team,
-            job_path=path,
-            reason=reason,
-            vdk_version=None,
-            enabled=True,
-        )
-        return (
-            f"Job with name {name} and team {team} is deployed successfully! "
-            f"Deployment information:\n {printer.get_memory().strip()}"
-        )
+
+        try:
+            cmd = JobDeploy(RestApiUrlConfiguration.get_rest_api_url(), printer)
+            cmd.create(
+                name=name,
+                team=team,
+                job_path=destination_path,
+                reason=reason,
+                vdk_version=None,
+                enabled=True,
+            )
+            return (
+                f"Request to deploy job with name {name} and team {team} is sent successfully!"
```

Comment:
Why not reuse the full description like in the CLI :

                f"Request to deploy Data Job {name} using version {deployment.job_version} finished successfully.\n"
                f"It would take a few minutes for the Data Job to be deployed in the server.\n"
                f"If notified_on_job_deploy option in config.ini is configured then "
                f"notification will be sent on successful deploy or in case of an error.\n\n"

Reply:
added the missing information